core/mondoo-gcp-inventory.mql.yaml

```diff
@@ -0,0 +1,85 @@
+packs:
+  - uid: mondoo-asset-inventory-gcp
+    name: Mondoo mondoo-asset-inventory-gcp Asset Inventory
+    tags:
+      mondoo.com/platform: gcp,cloud
+      mondoo.com/category: best-practices
+    docs:
+      desc: |
+        The Mondoo GCP Inventory query pack gathers information about GCP projects for asset inventory.
+    filters:
+      - asset.platform == "gcp"
+      - uid: mondoo-asset-inventory-gcp-project-info
+        title: Gather GCP Project Information
+        query: gcp.project { name id number state labels }
+      - uid: mondoo-asset-inventory-gcp-project-owners
+        title: Gather data for all owners of the GCP project
+        docs:
+          desc: |
+            This query gathers data for all owners of the GCP project
+        query: |
+          gcp.project.iamPolicy.where( role == "roles/owner" ) { * }
+      - uid: mondoo-asset-inventory-gcp-project-editors
+        title: Gather data for all editors of the GCP project
+        docs:
+          desc: |
+            This query gathers data for all editors of the GCP project
+        query: |
+          gcp.project.iamPolicy.where( role == "roles/editors" ) { * }
+      - uid: mondoo-asset-inventory-gcp-iam-roles
+        title: Gather list of all IAM Policy roles for the GCP project
+        docs:
+          desc: |
+            This query gathers a list of all roles defined for a GCP project
+        query: |
+          gcp.project.iamPolicy { role }
+      - uid: mondoo-asset-inventory-gcp-enabled-services
+        title: Gather all services enabled in the GCP project 
+        docs: 
+          desc: |
+            This query gathers a list of all services enabled in the GCP Project
+        query: gcp.project.services.where( enabled == true ) { * }
+      - uid: mondoo-asset-inventory-gcp-gke-clusters-count
+        title: Gather count of GKE clusters in a GCP project
+        docs: 
+          desc: |
+            This query gathers a count of GKE clusters running in a GCP project
+        query: gcp.project.clusters.length
```

Comment:
This was renamed to `gcp.project.gke.clusters`

Reply:
@chris-rock I just tested this with cnspec `7.11.0` and the query failed, and then upgraded to `7.12.0` and it worked. Are you ok with `gcp.project.gke.clusters` not being backwards compatible?

Reply:
Yes, we had `gcp.project.clusters` in evaluation and it turned out not to be the right place. The way to go is `gcp.project.gke.clusters`